docs/install/prerequisites.md

```diff
@@ -26,4 +26,22 @@ Before installation, you must meet the following prerequisites:
 
 - You have a cluster that uses Kubernetes v1.18 or newer.
 - You have installed the [`kubectl` CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
-- Your Kubernetes cluster must have access to the internet, since Kubernetes needs to be able to fetch images.
+- You have installed the [`kn` CLI](./install-kn). (See below)
```

Comment:
kn is optional and has nothing to do with installing any other features, it's not interchangeable with kubectl it just extends it to make it easier to create things like sources, channels, services, etc. I don't think it belongs in prereqs at all.

Reply:
While it isn't a hard requirement, it certainly enhances the Knative experience. 

While we often default to YAML, we should try and get people to interact with the CLI as often as possible. 

+ making this a pre-requisite means we don't have a weird "Next Steps" section with a `kn` CLI as an optional download, and we get people familiar with the idea that there is a way to interact with Knative directly.